modules/recommended-configurable-storage-technology.adoc

```diff
@@ -91,10 +91,13 @@ recommended for {product-title} Registry cluster deployment with production work
 
 In a scaled/HA {product-title} registry cluster deployment:
 
-* The preferred storage technology is object storage. The storage technology must support RWX access mode and must ensure read-after-write consistency.
-* File storage and block storage are not recommended for a scaled/HA {product-title} registry cluster deployment with production workloads.
-* All NAS storage are not recommended for {product-title} Registry cluster
-deployment with production workloads.
+* The preferred storage technology is object storage. The storage technology must
+support RWX access mode and must ensure read-after-write consistency.
```

Comment:
'read after write' is not a 'storges' [consistency term](https://en.wikipedia.org/wiki/Consistency_model). 

This was a term, that AWS (the service provider) coined with a the [this announcement of AWS S3](https://aws.amazon.com/about-aws/whats-new/2015/08/amazon-s3-introduces-new-usability-enhancements/). 

The point of this is, asking about service, outside of 'object' storage, to meet, have or offer this requirement, is absurd.

Reply:
The problem is object storage is available only for customers on cloud.
What about for customers with on-premise installation (vSphere, bare-metal)? File storage is the only option?
So we can't say file storage is not recommended for such customer.
We need a clear guidance, about the storage supportability for the internal registry.